### ISSUE TITLE
What is the use of masks in rpn/loss.py?

### ISSUE THREAD

**Issue by SkeletonOne** [krumo/Domain-Adaptive-Faster-RCNN-PyTorch#19]
Hi, I noticed that in rpn/loss.py, prepare_targets you have an output named "masks", but in __call__, you only do "torch.cat" to it and nothing else. So I am wondering what is the use of it? Thanks!

**Comment by krumo** [comment]
Hi @SkeletonOne,
thanks for this question. The masks here is useless. I avoid computing regression losses on target domain by avoiding generating bbox labels for target domain instead of using a mask to filter out target domain losses. There is some redundancy in the current implementation and I will fix it in the later update. Thanks again for pointing this problem out!

**Comment by Feobi1999** [comment]
which part of your code means “ I avoid computing regression losses on target domain by avoiding generating bbox labels for target domain instead of using a mask to filter out target domain losses.” Beacause I didnot find the related process in rpn.py?
Thanks
